android/app/build.gradle

```diff
@@ -202,6 +204,8 @@ android {
 }
 
 dependencies {
+    implementation project(':react-native-permissions')
```

Comment:
I'm a little confused as to why we need to include this dependency - how are we currently requesting permissions through `react-native-permissions` without having this setup?

Reply:
I am not sure about your current setup. I was just following the installation [instructions for react-native-qrcode-scanner](https://github.com/moaazsidat/react-native-qrcode-scanner#to-install-and-start-using-react-native-qrcode-scanner). This code was added automatically by running `react-native link react-native-permissions`.

Reply:
Interesting. This looks right to me, I'm more just confused by the fact that we already have `react-native-permissions` as an NPM dependency, and we're already getting location permissions from users without this line. 

Moreso just confused as to how this was working for Location permissions without this line 🤷‍♂️